src/utils/index.js

```diff
@@ -217,11 +217,18 @@ export function pickerOptionsFromAccept(accept) {
 
         return ok;
       })
-      .map(([mimeType, ext]) => ({
-        accept: {
+      .reduce(
+        (agg, [mimeType, ext]) => ({
```

Comment:
args?

Reply:
Short for aggregate, usually the name I give to the first argument of reduce function.  
Want me to change it to something else?

Reply:
Makes sense, please merge ASAP. Would be greatly appreciated.